lib/translator/inquiry.h

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef LIB_TRANSLATOR_INQUIRY_H
+#define LIB_TRANSLATOR_INQUIRY_H
+
+#include <cstdio>
+
+#include "absl/types/span.h"
+#include "common.h"
+#include "lib/scsi_defs.h"
+#include "third_party/spdk_defs/nvme_defs.h"
+#include "translation.h"
+
+namespace translator {
+
+StatusCode InquiryToNvme(absl::Span<const uint8_t> scsi_cmd,
+                         nvme_defs::GenericQueueEntryCmd &identify_ns,
+                         nvme_defs::GenericQueueEntryCmd &identify_ctrl,
+                         uint32_t &alloc_len);
+
+StatusCode InquiryToScsi(
+    absl::Span<const uint8_t> raw_cmd, absl::Span<uint8_t> buffer,
+    absl::Span<const nvme_defs::GenericQueueEntryCmd> nvme_cmds);
+
+// specific translation functions -- takes in nvme data and writes scsi result
+// to buffer
+void TranslateStandardInquiry(
```

Comment:
nit: other PRs have set the precedent for defining helper functions in anonymous namespaces so they are not exposed as part of a potential interface. Would it be appropriate here to do the same? As an overall note, I like where this PR has ended up!

Reply:
i'm happy how everything is coming together! i'm rewriting tests to be more end-to-end and will make an anon namespace